src/echarts.js

```diff
@@ -1142,8 +1143,26 @@ echartsProto.resize = function (opts) {
     flushPendingActions.call(this, silent);
 
     triggerUpdatedEvent.call(this, silent);
+
+    moveTooltip.call(this);
 };
 
+/**
+ * when alwaysShowContent is true change or rotation window size and restore will move tooltip
+ * @private
+ */
+function moveTooltip() {
+    var tooltips = filter(this._componentsViews, function (item) {
+        return item.__model.mainType === 'tooltip';
+    });
+    each(tooltips, function (tooltip) {
+        var realLeft = tooltip._tooltipContent._styleCoord[2] * this._zr.getWidth();
+        var realRight = tooltip._tooltipContent._styleCoord[1];
+        var ratio = tooltip._tooltipContent._styleCoord[2];
+        tooltip._tooltipContent.moveTo(realLeft, realRight, ratio);
```

Comment:
I can't find the usage of the parameter `ratio`, did I miss something?

Reply:
> I can't find the usage of the parameter `ratio`, did I miss something?

The code has just been updated to remove this parameter.